.rubocop.yml

```diff
@@ -51,6 +51,9 @@ Layout/ClassStructure:
 Layout/TrailingWhitespace:
   AllowInHeredoc: true
 
+Layout/LineLength:
+  Max: 120
+
```

Comment:
Can you update .rubocop_todo.yml, not .rubocop.yml in this PR?

```diff
--- a/.rubocop_todo.yml
+++ b/.rubocop_todo.yml
@@ -20,4 +20,4 @@ Metrics/MethodLength:
 # Configuration parameters: AutoCorrect, AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
 # URISchemes: http, https
 Layout/LineLength:
-  Max: 100
+  Max: 120
```

And other changes looks good to me. Can you squash your commits into one?

Reply:
@koic Done